client/dashboard/components/page-header/style.scss

```diff
@@ -0,0 +1,39 @@
+@import "@wordpress/base-styles/variables";
+
+.client-dashboard-components-page-header-heading {
+	font-size: $font-size-2x-large;
+	line-height: $font-line-height-2x-large;
+	margin-block: 0;
+}
+
+.client-dashboard-components-page-header-decoration {
```

Comment:
@jameskoster the renaming of the css classes wasn't intentional for some reason, right? I'll update them again.

Reply:
Oops, apologies!